### PR TITLE
Admin UI: redesigned signout page

### PR DESCRIPTION
**Pull request by Vultraz**
The signout page has always looked like a WIP design, so I redid it.

While signing out:
image

After signing out:
image

### CODE REVIEW

File: packages/app-admin-ui/client/pages/Signout.js [hunk 1]
```diff
@@ -86,24 +70,28 @@ const SignedOutPage = () => {
...
+          <Caption>Signing out.</Caption>
+        </Fragment>
+      ) : (
+        <Fragment>
+          <Animation name="pulse" duration="500ms">
+            <CheckIcon css={{ color: colors.primary, height: '3em', width: '3em' }} />
+          </Animation>
+          <Caption>You have been signed out.</Caption>
+          <FlexBox>
+            {/* TODO: is it ok to use '/' as a path here? */}
```
Comment (MadeByMike) [inline packages/app-admin-ui/client/pages/Signout.js]: Not necessarily, they might not even have a front-end application.

Comment (MadeByMike) [reply packages/app-admin-ui/client/pages/Signout.js]: I'd remove this as we can't make and assumption and don't want to cross concerns between apps

**Comment by MadeByMike** [comment]
I'd remove the home link ... although I would like a better way for sharing information across the 'app stack' I'd love for the adminUI to be able to query something and get the base URL. Similarly we need a better way for sharing graphQL end points with the AdminUI currently it's in the initialisation of the admin app and only 1.

File: packages/app-admin-ui/client/pages/Signout.js (hunk 1 above)
Comment (Vultraz) [reply packages/app-admin-ui/client/pages/Signout.js]: What about a UI hook (and if so, should a Home button be included or excluded by default)?

**Comment by Vultraz** [comment]
@MadeByMike I commented out the Home button for now. I'll deal with adding a UI hook separately.

**Comment by MadeByMike** [comment]
@Vultraz could you remove it? I don't like merging commented out functional code. Issues are the right to document this if you think it needs to be captured as as feature.

File: packages/app-admin-ui/client/pages/Signout.js [hunk 2]
```diff
@@ -86,24 +70,29 @@ const SignedOutPage = () => {
...
+          <Animation name="pulse" duration="500ms">
+            <CheckIcon css={{ color: colors.primary, height: '3em', width: '3em' }} />
+          </Animation>
+          <Caption>You have been signed out.</Caption>
+          <FlexBox>
+            {/* TODO: handle with UI hook
+            <SignOutPageButton appearance="primary" href="/">
+              Home
+            </SignOutPageButton>
+            */}
```
Comment (MadeByMike) [inline packages/app-admin-ui/client/pages/Signout.js]:
```suggestion
```

**Comment by MadeByMike** [comment]
I've remove it :)

**Comment by Vultraz** [comment]
@MadeByMike anything else I should do on this?

**Comment by timleslie** [comment]
@MadeByMike Could you re-review this now?

**Pull request merged**